src/python/pants/bin/remote_pants_runner.py

```diff
@@ -156,34 +310,23 @@ def _connect_and_execute(self, port):
     # Exit.
     self._exiter.exit(result)
 
-  def _extract_remote_exception(self, pantsd_pid, nailgun_error):
-    """Given a NailgunError, returns a Terminated exception with additional info (where possible).
-
-    This method will include the entire exception log for either the `pid` in the NailgunError, or
-    failing that, the `pid` of the pantsd instance.
-    """
-    sources = [pantsd_pid]
-    if nailgun_error.pid is not None:
-      sources = [abs(nailgun_error.pid)] + sources
-
-    exception_text = None
-    for source in sources:
-      log_path = ExceptionSink.exceptions_log_path(for_pid=source)
-      exception_text = maybe_read_file(log_path, binary_mode=False)
-      if exception_text:
-        break
-
-    exception_suffix = '\nRemote exception:\n{}'.format(exception_text) if exception_text else ''
-    return self.Terminated('abruptly lost active connection to pantsd runner: {!r}{}'.format(
-      nailgun_error, exception_suffix))
-
   def _restart_pantsd(self):
     return PantsDaemon.Factory.restart(bootstrap_options=self._bootstrap_options)
 
   def _maybe_launch_pantsd(self):
     return PantsDaemon.Factory.maybe_launch(bootstrap_options=self._bootstrap_options)
 
   def run(self, args=None):
-    self._setup_logging()
+    # Redirect fatal error logging to the current workdir, set the stream to log stacktraces to on
```

Comment:
all of this can be reverted since we are removing `RemoteExiter`etc from this change.

Reply:
This file has been reverted entirely!